### PR TITLE
SRCH-2587 add translations from Punchcard

### PR DESCRIPTION
**Pull request by MothOnMars**
## Summary
This PR:
- adds all the translations files that have until now been stored in [Punchcard](https://github.com/GSA/punchcard/tree/master/localizations)
- removes unused translations
- enables the `i18n-tasks` spec for missing translations
- updates the tests to reflect the updated translations 
- notes the Jira tickets required to enable the remaining `i18n-tasks` specs
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:

#### Functionality Checks

- [x] Code is functional. - basic smoke tests of English, Spanish, & French, & Russian pass.

- [x] Automated checks pass, if applicable. If Code Climate checks do not pass, explain reason for failures:

- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock.
 
- [x] You have merged the latest changes from the target branch (usually `master` or `main`) into your branch.
 
- [x] If your target branch is NOT `master` or `main`, specify the reason:
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release #.#.#** matching the release number
 
- [x] You have squashed your commits into a single commit (exception: your PR includes commits with formatting-only changes, such as required by Rubocop). - This PR includes separate commits to clarify what changes are happening at each step. The vast majority of changes are in the initial commit ([3c8d5cb](https://github.com/GSA/search-gov/commit/3c8d5cbf37dd1d22a8b06ed414228ccad8c29438)), which is a straight-up copy and paste of all the translations files in Punchcard: https://github.com/GSA/punchcard/tree/master/localizations. The subsequent commits clean up those files and make the required spec & config changes.

- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

File: config/cucumber.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <%
 rerun = File.file?('rerun.txt') ? IO.read('rerun.txt') : ""
 rerun_opts = rerun.to_s.strip.empty? ? "--format #{ENV['CUCUMBER_FORMAT'] || 'progress'} features" : "--format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} #{rerun}"
-std_opts = "--format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} --strict --tags 'not @wip' --guess"
+std_opts = "--format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} --strict --tags 'not @wip' --guess --publish-quiet"
```
Comment (MothOnMars) [inline config/cucumber.yml:4]: turn off a noisy Cucumber message

File: spec/models/navigable_name_updater_spec.rb [hunk 2]
```diff
@@ -14,8 +13,13 @@
...
 
-    it 'updates all video search labels except for English/Spanish sites' do
-      expect { navigable_name_updater.update }.to change { RssFeed.where(name: 'Test Entry').count }.from(0).to(1)
+    context 'when a video feed label is outdated' do
+      before { video_feed.update!(name: 'outdated') }
+
+      it 'updates all video search labels except for English/Spanish sites' do
+        expect { navigable_name_updater.update }.
+          to change { video_feed.reload.name }.from('outdated').to('Videos')
+      end
```
Comment (MothOnMars) [inline spec/models/navigable_name_updater_spec.rb:22]: I've updated this spec minimally to make it pass. I don't love the update, but it was a weird spec to begin with. The code itself is walking dead, so I opted not to burn any more time on refactoring the spec.

**Comment by MothOnMars** [comment]
@jmax-fearless , this won't be officially ready for review until https://github.com/GSA/search-gov/pull/815 is in master, as these changes depend on that PR. So this draft PR temporarily includes those changes. Per our discussion, this PR will include separate commits to clarify what changes are happening at each step. The vast majority of changes are in the initial commit (https://github.com/GSA/search-gov/pull/820/commits/3c8d5cbf37dd1d22a8b06ed414228ccad8c29438), which is a straight-up copy and paste of all the translations files in Punchcard: https://github.com/GSA/punchcard/tree/master/localizations. The subsequent commits clean up those files and make the required spec & config changes. 

Feel free to ignore this until it's ready for review, but I thought you might want to give it a quick look to see if it all  makes sense.

**Review by MyNameIsMissing — APPROVED** [review]
Team based review and now my review just trying to learn a bit.
Also previously reviewed by John while in draft.
LGTM

**Comment by MothOnMars** [comment]
@MyNameIsMissing , here's one more PR for re-review, based on one additional commit: https://github.com/GSA/search-gov/pull/820/commits/d8e72cf6434c2d75c080a15c52c96ff67106b757

That commit copies over the spec files being removed from Punchcard (which was a complete oversight on my part): https://github.com/GSA/punchcard/pull/51#issuecomment-964384973

There are a number of new Code Climate failures caused by the addition of the legacy Punchcard specs: https://codeclimate.com/github/GSA/search-gov/pull/820. I cleaned up a few Rubocop offenses, but determined that deeper refactoring is out of scope for this story. If you agree, we can manually approve the Code Climate failures.

**Review by MyNameIsMissing — APPROVED** [review]
Note that I feel we should address the `Avoid instance variables` issues whenever we have other reasons to refactor.  The other CodeClimate items seem trivial.  

LGTM

**Pull request merged**